src/iosrv/tests/drpc_handler_tests.c

```diff
@@ -49,9 +49,15 @@ dummy_drpc_handler2(Drpc__Call *request, Drpc__Response **response)
 {
 }
 
+static void
```

Comment:
What do you mean?

Reply:
Sorry for being unclear. I didn't see any other changes in this file, so I was wondering if there were test changes in progress that use this function? Otherwise it is not needed.